contracts/identity/lib.rs

```diff
@@ -94,14 +103,17 @@ mod identity {
 
 	/// Storage
 	#[ink(storage)]
-	#[derive(Default)]
 	pub struct Identity {
 		number_to_identity: Mapping<IdentityNo, IdentityInfo>,
 		owner_of: Mapping<IdentityNo, AccountId>,
 		identity_of: Mapping<AccountId, IdentityNo>,
 		identity_count: u32,
+		network_id_to_name: Mapping<NetworkId, String>,
+		network_name_to_id: Mapping<String, NetworkId>,
```

Comment:
I just wanted to prevent malicious actors from inserting the same network twice.
But as this is a admin-only method, it makes sense to remove the `network_name_to_id` mapping.